src/utilities/buildASTSchema.js

```diff
@@ -441,6 +447,7 @@ export class ASTDefinitionBuilder {
           name,
           description,
           astNode,
+          specifiedBy: getSpecifiedBy(astNode),
```

Comment:
Please make sure that this also works in `extendSchema`:
```gql
extend scalar Foo @specified(by: 'https://example.com/foo_spec')
```

Reply:
I added [this test](https://github.com/graphql/graphql-js/pull/2276/files#diff-c766829f03b82fbf1ebc1ea0926dd286R341-R368), but was unsure about the preferred implementation when there were multiple extensions (some of which could be unrelated to `specifiedBy`).

[Right now](https://github.com/graphql/graphql-js/pull/2276/files#diff-a2222d77ff884acdf97c5f295babf27cR330-R343) I am creating an array with the base `.specifiedBy` value, and calling `getSpecifiedBy` on each extension, filtering out the undefined values with `.filter(Boolean)`, and then getting the "last value".  With this if there are no `specifiedBy` values, it will try to get the `-1` element of the array, which is always `undefined`, and valid, but feels a bit weird.

Open to suggestions.